src/wp-admin/includes/class-wp-filesystem-direct.php

```diff
@@ -316,7 +316,14 @@ public function copy( $source, $destination, $overwrite = false, $mode = false )
 	}
 
 	/**
-	 * Moves a file.
+	 * Moves a file or directory.
+	 *
+	 * After moving files or directories, OPcache will need to be invalidated.
+	 *
+	 * If moving a directory fails, `copy_dir()` can be used for a recursive copy.
+	 *
+	 * Use `move_dir()` for moving directories with OPcache invalidation and a
+	 * fallback to `copy_dir()`.
```

Comment:
Once this is merge ready, let's add the same docs changes to the other classes.

Parameters will need to be updated to change file to "file or directory."

Reply:
Good point Peter. I've posted this onto the ticket for posterity.